kotlin-sdk/src/commonMain/kotlin/dev/openfeature/kotlin/sdk/OpenFeatureAPIInstance.kt

```
@@ -207,13 +219,70 @@ open class OpenFeatureAPIInstance internal constructor() {
+                    activeContextReconciliations = 0
+                }
+                activeContextReconciliations++
+                reconciliationRegistered = true
+                if (activeContextReconciliations == 1) {
+                    _statusFlow.emit(OpenFeatureStatus.Reconciling)
+                }
+            }
+
+            reconciliation.provider.onContextSet(reconciliation.oldContext, evaluationContext)
+            terminalStatus = OpenFeatureStatus.Ready
+        } catch (e: CancellationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)